core/src/main/java/com/flipkart/gjex/core/task/RotationManagementBasedHealthCheck.java

```diff
@@ -0,0 +1,54 @@
+package com.flipkart.gjex.core.task;
+
+import com.flipkart.gjex.core.logging.Logging;
+import io.dropwizard.metrics5.health.HealthCheck;
+
+import javax.inject.Singleton;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Rotation management based health check for the app
+ * @author ajay.jalgaonkar
+ */
+
+@Singleton
+public class RotationManagementBasedHealthCheck extends HealthCheck implements Logging {
+  private static final String BIR = "bir";
+  private static final String OOR = "oor";
+
+  private AtomicBoolean state;
```

Comment:
Can we refactor state attribute to maybe inRotation to signify what it will actually store .

Reply:
refactored to rotationStatus